swag/db.py

```diff
@@ -44,6 +58,15 @@ def __init__(
         self.blockchain = blockchain
         self.guild_timezone = guild_timezone
 
+        self.next_cagnotte_id = next_cagnotte_id
+        self.cagnotte_id = cagnotte_id
```

Comment:
If we can't access a Cagnotte with its name therefore this `dict` is useless

Reply:
Resolved here https://github.com/BobbySoftwares/Bybbot/pull/7/commits/50df69e8dbcb5935bd5849b53157b42661ba1c65 and here https://github.com/BobbySoftwares/Bybbot/pull/7/commits/8455c48be74193cc6dda0089c51fecbed020817c